src/sonic-yang-models/setup.py

```diff
@@ -55,6 +55,13 @@
                          './yang-models/sonic-types.yang',
                          './yang-models/sonic-versions.yang',
                          './yang-models/sonic-vlan.yang',
+                         './yang-models/sonic-dscp-tc-map.yang',
+                         './yang-models/sonic-dot1p-tc-map.yang',
+                         './yang-models/sonic-tc-dscp-map.yang',
+                         './yang-models/sonic-tc-dot1p-map.yang',
+                         './yang-models/sonic-tc-priority-group-map.yang',
+                         './yang-models/sonic-tc-queue-map.yang',
+                         './yang-models/sonic-pfc-priority-queue-map.yang',
```

Comment:
I don't see yang model for PORT_QOS_MAP (unless I missed it). Is there a separate PR for it.

![image](https://user-images.githubusercontent.com/10645050/134988532-605282cb-9cb0-4f7c-9eec-ffa205e4d781.png)

Reply:
@smaheshs - We have not done this because PORT_QOS_MAP is reference to maps. I will raise it once format changes are merged.